docs/sources/features/datasources/elasticsearch.md

```diff
@@ -91,6 +91,15 @@ For example, if you're using a default setup of Filebeat for shipping logs to El
 - **Message field name:**  message
 - **Level field name:** fields.level
 
+### Data links
```

Comment:
This is going to be very confusing, considering we have a feature of the same name that works differently than this. @dprokop, thoughts?

Reply:
This feature is actually fairly similar and we will probably unify it even more in the future so not sure if it make sense to change the name right now. There is a difference in with Loki where we do data link + create a new field that's why the naming there is different.